### DISCUSSION TITLE
How to test crashlytics on iOS?

### DISCUSSION THREAD

**Discussion by zverexe** [invertase/react-native-firebase#5749] [Q&A] [answered]
I'm trying to set crashlytics on ios and emulate app crash in order to start app crash dashboard in firebase console. I checked different options and found this PR https://github.com/invertase/react-native-firebase/issues/5053 but still, I'm unaware of how to simulate crash on ios. As it was mentioned here in discussions, crash on ios handled by Xcode debugger and firebase crashlytics doesn't fire. Is there any workaround on how I can run it?

**Comment by mikehardy** [comment, marked as answer]
Hi there @zverexe - run it via the command line, not via Xcode, you can just make a debug page that is only present in debug builds, enable crashlytics logging in debug via firebase.json, run pod install (so firebase.json is ingested in the plist later when you build), re-build and run it.

You can examine our tests directory in the repo to see how we do it

https://github.com/invertase/react-native-firebase/blob/47d226526fec4dcee6edc5f5c2e37f430e91cae9/tests/firebase.json#L11-L14

https://github.com/invertase/react-native-firebase/blob/47d226526fec4dcee6edc5f5c2e37f430e91cae9/tests/app.js#L63-L77

https://invertase.io/blog/react-native-firebase-crashlytics-configuration

**Reply by zverexe** [reply]
@mikehardy thank you!
